client/components/Nav.jsx

```diff
@@ -422,6 +422,7 @@ class Nav extends React.PureComponent {
                   onBlur={this.handleBlur}
                 >
                   Add File
+                  <span className="nav__keyboard-shortcut">{metaKeyName}+Shift+3</span>
```

Comment:
Thanks @nik72619c for the change, but I am still seeing the log statement there and one more thing the build is failing because some tests are failing can you check that too.